utilities/gsheets-import/resultsImporter.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/python
```

Comment:
I'd love this tool to be just a little more flexible. But that's covered by #370.